docs/help.rst

```diff
@@ -247,3 +247,29 @@ Alert Manager is not working
                             resources:
                               requests:
                                 storage: 10Gi
+
+
+CRD issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. details:: CustomResourceDefinition.apiextensions.k8s.io "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long
+
+      This is often a CRD issue which can be fixed with :ref:`these instructions <Manual Upgrade>` To verify this, open the ArgoCD error logs and look for more details.
```

Comment:
@pavangudiwada this is the wrong fix. You don't fix by applying a manual upgrade, you fix by checking off the "server-side apply" option

Reply:
(The image below is right, the text above is not.)

Reply:
https://blog.ediri.io/kube-prometheus-stack-and-argocd-25-server-side-apply-to-the-rescue

Reply:
Fixed it